commerce_coordinator/apps/commercetools/utils.py

```diff
@@ -164,6 +169,18 @@ def has_full_refund_transaction(payment: Payment):
     return False
 
 
+def find_refund_transaction(order: Order, amount: decimal):
+    """
+    Utility to find the refund transaction in a payment
+    """
+    pmt = get_edx_successful_stripe_payment(order)
+    for transaction in pmt.transactions:
+        if transaction.type == TransactionType.REFUND:
+            if decimal.Decimal(typed_money_to_string(transaction.amount, money_as_decimal_string=True)) == amount:
+                return transaction.id
```

Comment:
This seems like a line that we'd want coverage for to confirm the right thing (transaction.id) is being returned